imports/email-templates/components/helpers.js

```diff
@@ -15,13 +17,9 @@ function engage (params) {
   return URL.format(engagementURL)
 }
 
-// Does not work on localhost domains, since we assume 4 parts: $service.$stage.unee-t.com
 function resolveServiceDomain (service) {
-  const url = URL.parse(process.env.ROOT_URL)
-  const hparts = url.hostname.split('.')
-  hparts[0] = service
-  url.host = hparts.join('.')
-  return URL.format(url)
+  const domain = process.env.STAGE ? `${service}.${process.env.STAGE}.${process.env.DOMAIN}` : `${service}.${process.env.DOMAIN}`
```

Comment:
We haven't introduced STAGE and DOMAIN to master yet afaik

Reply:
Actually, I just checked, they are there.